src/js/views/renderCharacter.js

```diff
@@ -32,22 +32,22 @@ export const createHeroListElement = () => {
 };
 
 // eslint-disable-next-line max-len
-export const insertCharacterHTML = (heroNode, heroName, heroIntelligence, heroStrength, heroCombat) => {
-  heroNode.insertAdjacentHTML('beforeend', characterHTML(heroName, heroIntelligence, heroStrength, heroCombat));
+export const insertCharacterHTML = (heroNode, heroName, heroIntelligence, heroStrength, heroCombat, herosImage) => {
+  heroNode.insertAdjacentHTML('beforeend', characterHTML(heroName, heroIntelligence, heroStrength, heroCombat, herosImage));
 };
 
 export default function renderCharacters(arr) {
 // eslint-disable-next-line no-unused-vars
-  arr.forEach((element) => {
-    console.log(element);
+  arr.forEach(async (element) => {
+    // add hero name, stats, and image to card
     const heroName = element.name;
     const heroIntelligence = element.powerstats.intelligence;
     const heroStrength = element.powerstats.strength;
     const heroCombat = element.powerstats.combat;
```

Comment:
🏆 Nice work declaring (and naming!) these vars and then passing them in to your function!